input/gcamdata/inst/extdata/socioeconomics/GCAM3_GDP.csv

```diff
@@ -4,17 +4,17 @@
 # ----------
 region_GCAM3,1975,1990,2005,2010,2015,2020,2025,2030,2035,2040,2045,2050,2055,2060,2065,2070,2075,2080,2085,2090,2095,2100
-Africa,250238,372580,592628,731932,878165,1033420,1206410,1384040,1581860,1797360,2062330,2387100,2815650,3348790,4004760,4796430,5711680,6745020,7896200,9211340,10709700,12273400
-Australia_NZ,174747,260506,446803,490378,548937,610867,674627,742353,811550,887211,964484,1043320,1121750,1201990,1275340,1349670,1412750,1476960,1538550,1606720,1700620,1777130
-Canada,278868,440750,666479,690660,771776,845810,916883,985027,1055440,1131290,1211240,1304520,1396260,1500570,1605540,1725610,1851220,1984880,2129050,2293760,2488970,2682490
-China,118793,386370,1592970,2455650,3512670,4777640,6245180,7943570,9848140,11917000,14119100,16441400,18851100,21457500,24317700,27633900,31475400,35845200,40584400,45170900,48397800,52233700
-Eastern Europe,232364,284949,409212,466921,549213,635370,715629,808061,897352,1027510,1158100,1329870,1469020,1648870,1813000,2031990,2240430,2470430,2705010,3007460,3407960,3768550
-Former Soviet Union,410608,468434,427135,492322,587775,693188,803321,930896,1066830,1235350,1417930,1631650,1852670,2103660,2372560,2678560,3007550,3372570,3788340,4268530,4866440,5467570
-India,106889,218502,522833,746024,1027350,1382050,1842840,2404660,3097150,3922800,4903620,6055790,7400990,8909040,10578900,12414400,14394400,16538800,18849800,21522700,24635300,27860600
-Japan,1830700,3341320,4049050,3937880,4139010,4339740,4546610,4766620,4990470,5250430,5512880,5798520,6080380,6373160,6672850,6967690,7243710,7534980,7829250,8160520,8568700,8925240
-Korea,71339,230019,538839,607069,656741,707030,756512,810748,870628,941268,1017410,1099640,1187740,1282620,1378260,1475150,1570990,1668740,1766150,1880310,2024860,2151700
-Latin America,814694,1187430,1919720,2234190,2553590,2949630,3368380,3834340,4299980,4873900,5476220,6236520,7014730,8019950,9193280,10738400,12553700,14719600,17144300,19760200,22336300,25366900
-Middle East,298287,354600,649161,799322,974253,1174110,1392710,1635480,1889650,2189190,2499060,2852530,3190200,3578500,3967850,4432780,4896340,5423610,5952670,6525370,7119600,7769320
-Southeast Asia,222332,593705,1230630,1460320,1961080,2601350,3372790,4305090,5403760,6689470,8210530,9959920,12061300,14448000,17201800,20092200,23375600,26794100,30544100,34509200,38874800,43534900
-USA,3473910,5728530,9104140,9600970,10756900,11914400,13175100,14458700,15829500,17465700,19164900,21251000,23198700,25517400,27872600,30669200,33583600,36744000,39970500,43838800,48155500,52760200
-Western Europe,3802400,5646240,7801260,7930550,8546440,9184410,9843660,10470000,11052000,11624200,12189200,12795600,13407800,14089100,14736400,15486200,16180100,16919800,17638600,18551700,19738200,20254600
+Africa,250238,372580,2285502,2935806,3471964,3806228,4649023,5868433,7687257,10008768,12969888,16749961,21576781,27668493,35171412,44298418,55210792,68055680,82916939,99846450,118969449,140339237
```

Comment:
some of these changes (like the Africa GDP in 2005) are really big from the old file...any idea why? mostly concerned this may be a units issue

Reply:
Yes, I saw it, but since other regions have similar values to the old file I don't think it's a matter of units... But I'll recheck again just in case